R/pkg/R/DataFrame.R

```diff
@@ -65,6 +65,49 @@ dataFrame <- function(sdf, isCached = FALSE) {
 
 ############################ SparkDataFrame Methods ##############################################
 
+#' Return an SparkDataFrame's name.
```

Comment:
you probably want to add a line like
```
#' Name of SparkDataFrame
```

The first line is the title of the help page, an empty line (`#'`) and the next line is the description:
```
#' Title
#'
#' Description
```